crates/uv/src/commands/run.rs

```diff
@@ -98,6 +103,23 @@ struct RunEnvironment {
     _temp_dir_drop: Option<TempDir>,
 }
 
+fn find_workspace_requirements() -> Result<Option<RequirementsSource>> {
+    // TODO(zanieb): Add/use workspace logic to load requirements for a workspace
+    // We cannot use `Workspace::find` yet because it depends on a `[tool.uv]` section
```

Comment:
cc @charliermarsh minor note for consideration in your design

Reply:
Yes definitely, just not implementing it yet. I was hoping I could re-use some logic in the `uv-workspace` crate.